development/postgresdb.tf

```diff
@@ -7,7 +7,7 @@ module "postgres_db_development" {
   vpc_id =  "vpc-0d15f152935c8716f"
   db_engine = "postgres"
   db_engine_version = "16.3"
-  db_parameter_group_name = "postgres-16"
+  db_parameter_group_name = "postgres16"
```

Comment:
This change wasn't needed as the `postgres16` did the exact same thing:
![image](https://github.com/user-attachments/assets/58144b89-740d-46fd-9da6-9b8ffa8ad7e7)

Reply:
Seems like we have 2 parameter groups with the same configuration.
Will need to decide which one to keep:
![image](https://github.com/user-attachments/assets/c0339452-7e29-4e4b-890c-7f0ffba1a2ff)

Reply:
Yes I think some of our work overlapped here.  Agreed one will have to be removed as they both do the same thing.